dev/doc/changes.md

```diff
@@ -4,6 +4,12 @@
   comes from a notation. Use `None` if not and `Some foo` to tell to
   print such TacGeneric surrounded with `foo:( )`.
 
+### Code formatting
+
+- The automatic code formatting tool `ocamlformat` has been disabled and its
+  git hook removed. If desired, automatic formatting can be achieved by calling
+  the `fmt` target of the dune build system.
```

Comment:
This claim is inconsistent with the removal of the `.ocamlformat` files that is still part of this PR. With this removal, this target will do nothing.